plugins/devtools-backend/src/service/router.ts

```diff
@@ -48,6 +54,8 @@ export async function createRouter(
 ): Promise<express.Router> {
   const { logger, config, permissions } = options;
 
+  const { httpAuth } = createLegacyAuthAdapters(options);
```

Comment:
This one took me a bit to figure out trying to understand why `discovery` was required but `httpAuth` was not. Once I found the code for `createLegacyAuthAdapters` it made more sense 👍